### ISSUE TITLE
HyperlinkButton Accessibility control type cannot be overridden

### ISSUE THREAD

**Issue by JordanBluehdorn** [microsoft/microsoft-ui-xaml#9595]
### Describe the bug

Setting the AutomationProperties.AutomationControlType of a hyperlinkbutton has no effect on the narrator. It still reads as a hyperlink.

### Steps to reproduce the bug

1. Create a hyperlink button
2. Set the AutomationProperties.AutomationControlType to Button
3. Open Narrator
4. Observe behavior

### Expected behavior

Narrator should read the control as a "Button"

### Screenshots

Narrator reads the control as a "Hyperlink"

### NuGet package version

None

**Comment by RBrid** [comment]
Thanks for the report. We made a code change in WinUI3 two years ago (in FrameworkElementAutomationPeer::GetAutomationControlType) that is expected to address this, and indeed I could to repro the problem. I tested with:

```
    <StackPanel>
        <Button x:Name="myTestButton" Content="Hello" Margin="30"/>
        <HyperlinkButton x:Name='myHyperlinkButton' NavigateUri='http://www.msn.com' Content='M S N link' Margin='30'/>
        <RepeatButton x:Name="myTestRepeatButton" Content="Repeatedly" Margin="30"/>

        <Button x:Name="mySecondTestButton" Content="Hello Two" Margin="30, 60, 30, 30" AutomationProperties.AutomationControlType="Calendar"/>
        <HyperlinkButton x:Name='mySecondHyperlinkButton' NavigateUri='http://www.broadway.com' Content='Broadway link' Margin='30' AutomationProperties.AutomationControlType="Button"/>
        <RepeatButton x:Name="mySecondTestRepeatButton" Content="Repeatedly Two" Margin="30" AutomationProperties.AutomationControlType="DataGrid"/>
    </StackPanel>
```

...and all went well. Narrator uses the custom control types.
